pwiz_tools/Skyline/TestRunnerLib/RunTests.cs

```diff
@@ -172,6 +175,9 @@ private static string GetAssemblyPath(string assembly)
 
         public bool Run(TestInfo test, int pass, int testNumber, string dmpDir)
         {
+            if (TeamCityTestDecoration)
```

Comment:
Please use a function for all 3 cases where you use if (TeamCityTestDecoration) Console.WriteLine(). Thanks.

Reply:
You mean a function like Log() which is a no-op if TeamCityTestDecoration is false?

Reply:
Exactly, and which just takes the status and TestInfo object as parameters.

Reply:
And which does all the escaping.

Reply:
Makes your impact on a 200-line function just 3 added lines, right?

Reply:
Yes but I think it needs 2 functions. For start and end, where the end function takes an error message if the result is failure. Folding it into 1 function would require some kind of "message type" parameter or parsing the message itself.

Reply:
Looks good to me.